tests/PHPUnit/System/LabelFilterTest.php

```diff
@@ -29,6 +29,11 @@ public function testApi($api, $params)
         $this->runApiTests($api, $params);
     }
 
+    public static function setUpBeforeClass()
+    {
+        parent::setUpBeforeClass(); // TODO: Change the autogenerated stub
+    }
```

Comment:
Not super important to me, but don't need this method.

Reply:
will remove it 👍